pkg/postgres/query/query.go

```diff
@@ -71,3 +71,83 @@ func checkFieldValueType(field filter.RequestField) (valueIsList bool, valueList
 	}
 	return valueIsList, valueList, nil
 }
+
+func likeOperatorCondition(
+	field filter.RequestField, valueIsList bool, negate bool,
+) (conditionTemplate string, err error) {
+	quotedName, err := getQuotedName(field.Name)
+	if err != nil {
+		return "", errors.Wrap(err, "could not get quoted name")
+	}
+
+	if valueIsList {
+		valueList, ok := field.Value.([]any)
+		if !ok {
+			err = errors.New("couldn't not get field list values")
+			return
+		}
+		for _, element := range valueList {
+			if _, ok := element.(string); ok {
+				conditionTemplate = multiLikeOrTemplate(quotedName, len(valueList), negate)
```

Comment:
This does not seem to fit: `conditionTemplate` is nowhere stored and changed with each loop iteration. 

I assume it's enough to call it once, as the loop variable is not part of the statement

Reply:
Thanks, it is enough to call it just once. 
Moved the call outside the loop